src/app/newSettings/Sections/Account/Plans/PlansSection.tsx

```diff
@@ -310,7 +353,10 @@ const PlansSection = ({ changeSection }: PlansSectionProps) => {
                   : translate(`preferences.account.plans.${plan.interval}`)?.toLowerCase()
               }
               isCurrentPlan={
-                currentUserSubscription?.type === 'subscription' && currentUserSubscription?.priceId === plan.id
+                // currentUserSubscription?.type === 'subscription' && currentUserSubscription?.priceId === plan.id
+                selectedIntervalSubscription == 'individual'
```

Comment:
this comparison is quite repeated, maybe you could extract it to have it in one place.
e.g. `const isIndividualSubscriptionSelected = selectedIntervalSubscription == 'individual'`